signup.html

```diff
@@ -0,0 +1,135 @@
+            <ul class="nav-list">
+                <li><a href="index.html">Home</a></li>
+                <li><a href="shop.html">Shop</a></li>
+                <li><a href="">About</a></li>
+            </ul>
+
+            <!-- Login, Cart, Wishlist,  -->
+            <ul class="icons">
+                <li>
+                    <a href="#">
+                        <i class="search-icon btn-icon fa fa-search"></i>
+                    </a>
+                    <div class="search-bar search-form">
+                        <form>
+                            <input type="search" placeholder="Search" class="input-styled">
+                            <i class="btn-icon fas fa-search"></i>
+                        </form>
+                    </div>
+                </li>
+                <li>
+                    <a href="#">
+                        <i class="btn-icon fa fa-user"></i>
+                    </a>
+                </li>
+                <li>
+                    <a href="#">
+                        <i class="btn-icon fa fa-shopping-cart"></i>
+                    </a>
+                </li>
+                <li>
+                    <a href="#">
+                        <i class="btn-icon fa fa-heart"></i>
+                    </a>
+                </li>
+            </ul>
+
+            <!-- Burger -->
+            <div class="burger">
+                <div class="line1"></div>
+                <div class="line2"></div>
+                <div class="line3"></div>
+            </div>
+        </nav>
+
+        <!-- Signup container starts -->
+        <main class="signup__container">
+            <h3>SIGN UP</h3>
+            <div class="input-form">
+                <label class="label" for="email">Name<span class="red">*</span></label>
+                <input type="text" name="email" placeholder="Enter your name" required>
+                <label class="label" for="email">Email Id<span class="red">*</span></label>
+                <input type="text" name="email" placeholder="abc@gmail.com" required>
+                <label class="label" for="password">Password</label>
+                <input type="password" name="password" placeholder="*********" required>
+                <label class="label" for="password">Confirm Password</label>
+                <input type="password" name="password" placeholder="*********" required>
+
+                <div class="form-checks flex-align-center">
+                    <input type="checkbox" name="terms-conditions" />
+                    <label for="terms-conditions">
+                        <span>I agree to the <a href="#" class="btn-link-text">Terms and Conditions</a></span>
+                    </label>
+                </div>
+
+                <button class="btn btn-primary">LOGIN</button>
+                <a href="login.html" class="btn btn-link-text align-center"> Already registered? Log in<i class="fas fa-chevron-right"></i>
+                </a>
+                
```

Comment:
Just wrap this up in HTML `<form>` Tag. This will enable username and email field checks on button click.

Line 90 to 109

![image](https://user-images.githubusercontent.com/91944618/154801160-3d3b3b8c-91b0-4bf6-a5ee-f78243cb2e02.png)